src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1141,6 +1143,10 @@ class LeVADocumentUseCase extends DocGenUseCase {
     String createTRC(Map repo = null, Map data = null) {
         def documentType = DocumentType.TRC as String
 
+        def acceptanceTestData = data.tests.acceptance
```

Comment:
@metmajer  data can be null here, you sure this is save?

Reply:
The method's signature is misleading. TRC is now scheduled PRE_END of phase Test, where test results are injected always. Still missing: integration and unit tests.